go.sum

```diff
@@ -218,6 +218,8 @@ github.com/gogo/protobuf v1.2.1/go.mod h1:hp+jE20tsWTFYpLwKvXlhS1hjn+gTNwPg2I6zV
 github.com/gogo/protobuf v1.2.2-0.20190723190241-65acae22fc9d/go.mod h1:SlYgWuQ5SjCEi6WLHjHCa1yvBfUnHcTbrrZtXPKa29o=
 github.com/gogo/protobuf v1.3.1 h1:DqDEcV5aeaTmdFBePNpYsp3FlcVH/2ISVVM9Qf8PSls=
 github.com/gogo/protobuf v1.3.1/go.mod h1:SlYgWuQ5SjCEi6WLHjHCa1yvBfUnHcTbrrZtXPKa29o=
+github.com/golang-jwt/jwt v3.2.1+incompatible h1:73Z+4BJcrTC+KczS6WvTPvRGOp1WmfEP4Q1lOd9Z/+c=
```

Comment:
Why is `github.com/dgrijalva/jwt-go` still not removed from `go.sum`? Is it still an indirect dependency? If so, it may still pose a security threat.

Reply:
Hmm, good point, let me check.

Reply:
This comes as an indirect dependency from labstack/echo library:

```
go mod why github.com/dgrijalva/jwt-go
# github.com/dgrijalva/jwt-go
github.com/upbound/universal-crossplane/internal/upboundagent
github.com/labstack/echo/v4/middleware
github.com/dgrijalva/jwt-go
```

Unfortunately, that library does not have a version including a fix rather they claim not to be effected by the issue: https://github.com/labstack/echo/pull/1916#issuecomment-878046299

Opened another draft PR with replace however, it is failing with another issue during go mod tidy: https://github.com/upbound/universal-crossplane/pull/167

Not sure how to proceed here yet.

Reply:
My understanding from [that comment](https://github.com/labstack/echo/pull/1916#issuecomment-878046299) is that we are safe in this situation because `labstack/echo` claims no optional `audience` verification by default.  However, we had better keep an eye on `v5` of `labstack/echo` where they will get rid of the unmaintained `github.com/dgrijalva/jwt-go` dependency.  

Because `labstack/echo` is the only consumer of `github.com/dgrijalva/jwt-go` and because they claim no optional `audience` verification by default, I think we are good to go.

Reply:
And if somewhere, we were ourselves doing an optional `audience` verification using the vulnerable library (`github.com/dgrijalva/jwt-go`), then it would have been our direct dependency.

Reply:
@negz, @daren-eth, we may keep an eye on `labstack/echo` and `github.com/dgrijalva/jwt-go` as I had previously observed that we depend on them elsewhere, e.g., `upbound-api`.

Reply:
Thanks @turkenh for the investigation.

Reply:
Created this issue to track: https://github.com/upbound/universal-crossplane/issues/168